docs/src/tutorials/device-monitoring.md

```diff
@@ -0,0 +1,119 @@
+# Monitor your device with collectd
+
+With thin-edge.io device monitoring, you can collect metrics from your device
+and forward these device metrics to IoT platforms in the cloud. 
+
+Using these metrics, you can monitor the health of devices
+and can proactively initiate actions in case the device seems to malfunction.
+Additionally, the metrics can be used to help the customer troubleshoot when problems with the device are reported.
+
+Thin-edge.io uses the open source component [`collectd`](https://collectd.org/) to collect the metrics from the device.
+Thin-edge.io translates the collected metrics from their native format to the [thin-edge.io JSON](../architecture/thin-edge-json.md) format
+and then into the [cloud-vendor specific format](../architecture/mapper.md).
+
+Enabling monitoring on your device is a 3-steps process:
+1. [Install `collectd`](#install-collectd),
+1. [Configure `collectd`](#configure-collectd),
+1. [Enable thin-edge.io monitoring](#enable-thin-edge-monitoring).
+
+## Install `collectd`
+
+Device monitoring is not enabled by default when you install thin edge.
+You will have to install and configure [`collectd`](https://collectd.org/) first.
+
+To install collectd, follow the [collectd installation process](https://collectd.org/download.shtml)
+that is specific to your device. On a Debian or Ubuntu linux:
+
+``` shell
+sudo apt-get install collectd-core
+```
+
+## Configure `collectd`
+
+### TLDR; Just want it running
+
+Thin-edge.io provides a [basic `collectd` configuration](https://github.com/thin-edge/thin-edge.io/blob/main/configuration/contrib/collectd/tedge-collectd.conf)
+that can be used to collect cpu, memory and disk metrics.
+
+Simply copy that file to the main collectd configuration file and restart the daemon.
+
+``` shell
+sudo cp /etc/tedge/contrib/collectd/tedge-collectd.conf /etc/collectd/collectd.conf
```

Comment:
```suggestion
sudo cp /etc/tedge/contrib/collectd/collectd.conf /etc/collectd/collectd.conf
```
I've renamed it in PR #276 to avoid the users accidentally forgetting to rename it from `tedge-collectd.conf` to `collectd.conf`

Reply:
Fixed.